public/app/features/alerting/unified/components/receivers/form/ChannelSubForm.tsx

```diff
@@ -94,7 +94,7 @@ export function ChannelSubForm<R extends ChannelValues>({
 
   const onResetSecureField = (key: string) => {
     if (_secureFields[key]) {
-      const updatedSecureFields = { ...secureFields };
+      const updatedSecureFields = { ..._secureFields };
```

Comment:
Change here is required when a subform contains more than one encrypted field, prior to this change it would always reset the secure fields to the original object but excluding only the key we pass in to this function.